src/test/fuzz/deserialize.cpp

```diff
@@ -35,6 +39,15 @@ void initialize()
 {
     // Fuzzers using pubkey must hold an ECCVerifyHandle.
     static const auto verify_handle = MakeUnique<ECCVerifyHandle>();
+
+    // Enable logging. This avoids accumulating allocated memory in a global
+    // state (via m_msgs_before_open buffering) if LogPrintf is called when
+    // fuzzing. Log messages are written to standard output.
```

Comment:
> Log messages are written to standard output

Why?

Reply:
My reasoning is that any `LogPrintf` in the deserialization code is likely a bug or a mistake, and thus something the fuzzing operator would like to be aware of. I'm scratching my own itch here: as a fuzzing operator myself I sure would like to know at least! :)

From the OP:

> This has the added benefit of making the fuzzing operator aware of any log printing caused by fuzzing which is likely to be an anomaly in itself (in the general case).

Makes sense? :)

Reply:
Logging to stdout in the fuzzer is spammy, I think. It distracts from the normal fuzzer output.

Reply:
Ah, that is true for deserialization fuzzers. However, for validation fuzzers or net processing fuzzers, or any other high level fuzzer, log prints are no longer an anomaly.

Reply:
This PR is only making change to the deserialization fuzzers.

Reply:
Sorry, I missed that.